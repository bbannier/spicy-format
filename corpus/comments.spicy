# Just a comment.

## This is fine.
##
## More stuff here.

public type A = B;         # Should stay here.

public type A = unit {};   # Should stay here.
public type A = struct {}; # Should stay here.

function f() {}            # Should stay here.

if (True) # Should stay here.
	1;
