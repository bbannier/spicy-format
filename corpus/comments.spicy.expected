# Just a comment.

## This is fine.
##
## More stuff here.

#
const a = 1;

public type A = B; # Should stay here.

public type A = unit {}; # Should stay here.
public type A = struct {}; # Should stay here.

function f() {} # Should stay here.

if (True) # Should stay here.
    1;

 type X = unit {
    switch (a) {

        # Comment
        cond -> A: uint8;

        # Comment
    };
};
